src/worker/token.worker.ts

```diff
@@ -40,10 +61,15 @@ const messageHandler = async ({
         throw new Error(MISSING_REFRESH_TOKEN_ERROR_MESSAGE);
       }
 
-      fetchOptions.body = JSON.stringify({
-        ...body,
-        refresh_token: refreshToken
-      });
+      fetchOptions.body = useFormData
+        ? createFormData({
```

Comment:
```js
new URLSearchParams({
  ...body,
  refresh_token: refreshToken
})
```

(double check Safari support and that `.filter(k => typeof params[k] !== 'undefined')` from `createFormData` is not required)